FiveCalls/FiveCalls/LocationSheet.swift

```diff
@@ -108,9 +108,24 @@ struct LocationSheet: View {
     }
     
     func locationSearch() {
-        _ = NewUserLocation(address: locationText) { loc in
-            store.dispatch(action: .SetLocation(loc))
-            dismiss()
+        locationError = nil
+
+        Task {
```

Comment:
These tasks can probably be refactored into an action and handled by middleware unless we want to keep them here because they are specific to this view only.

Reply:
No strong feeling on it as long as it's only being used here